client/webserver/site/src/js/locales.ts

```diff
@@ -224,7 +239,22 @@ export const enUS: Locale = {
   [ID_CONNECT_WALLET_ERR_MSG]: 'Failed to connect {{ assetName }} wallet: {{ errMsg }}',
   [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'Redeemed by {{ makerAddr }}',
   [ID_REFUND_IMMINENT]: 'Will happen in the next few blocks',
-  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Refund will happen after {{ refundAfterTime }}'
+  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Refund will happen after {{ refundAfterTime }}',
+  [ID_AVAILABLE_C]: 'Available',
+  [ID_LOCKED_C]: 'Locked',
+  [ID_IMMATURE_C]: 'Immature',
+  [ID_SWAPPING]: 'Swapping',
+  [ID_BONDED]: 'Bonded',
+  [ID_LOCKED_BAL_MSG]: 'Total funds temporarily locked to cover the costs of your bond maintenance, outstanding orders, and necessary fees',
+  [ID_IMMATURE_BAL_MSG]: 'Funds coming your way shortly, waiting for confirmation on the blockchain',
```

Comment:
How about "Incoming funds awaiting confirmation".
"coming your way" is coloquial, "shortly" is subjective and mostly meaningless, and "on the blockchain" is superflous.